### ISSUE TITLE
Last Event ID

### ISSUE THREAD

**Issue by graywolf336** [alexandrevicenzi/go-sse#23]
Does this package automatically send messages to a client based on their last event id or is that up to the person using this package?

I am attempting to figure out how to implement this, but am a little unclear about it.

**Comment by alexandrevicenzi** [comment]
No, no message is stored, thus there's no way to magically resend.

You need to implement some logic on how to store messages and send it to a client.
You would also need to store some client information or decide what logic you would use to send to a new and/or returning client.

**Comment by graywolf336** [comment]
How do we get notified when a client connects and their last event id? I see where internally in the package it is stored, but I don’t see how to access when a client connects externally of the package internals...maybe I am missing something here.

**Comment by alexandrevicenzi** [comment]
That's supposed to be fixed by #16, it misses events to be registered.

So, at this moment, I think it would not be possible to do what you want. Once we fix #16 it would be possible.
